source/form.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-image" src="img/logos/logo.svg" width="33" height="38"
+        alt="Логотип Cat Energy картинка рыжего кота в плаще.">
+    </a>
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-text" src="img/logos/text-logo.svg" width="101" height="18"
+        alt="Логотип Cat Energy текстовая надпись.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню.</span>
+      </button>
+      <ul class="site-list">
+        <li class="site-list__item">
+          <a href="index.html">
+            Главная
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="catalog.html">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="site-list__item site-list__item--active">
+          <a>
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="form-headline">
+    <h1 class="form-headline__title">Подбор программы</h1>
+    <p class="form-headline__offer">Заполните анкету, и мы подберем<br>программу питания для
+      вашего кота</p>
+    <form class="form" action="https://echo.htmlacademy.ru/" method="post">
+      <fieldset class="form__field">
+        <label class="form__label" for="name">
+          Имя:*
+        </label>
+        <input class="form__input" type="text" placeholder="БАРСИК" name="cat-name" required id="name">
+        <label class="form__label" for="weight">
+          Вес(кг):*
+        </label>
+        <input class="form__input" type="number" placeholder="7" name="cat-weight" required id="weight">
+        <label class="form__label" for="age">
+          Возраст(лет):
+        </label>
+        <input class="form__input" type="number" name="cat-age" placeholder="7" id="age">
+      </fieldset>
+      <fieldset class="form__field">
+        <ul class="radio-button">
+          <li class="radio-button__item">
+            <label class="radio-checkbox-control">
```

Comment:
```suggestion
            <label class="radio-control">
```

ну раз ты ниже назвал чекбоксы - чекбокс контролом, то тут тогда уже нет смысла называть это радио-чекбокс-контролом, потому что это класс не для двух контролов, а только для радио теперь остался.